Python/Module.cpp

```diff
@@ -631,12 +649,33 @@ PyMODINIT_FUNC initPyHSPlasma() {
     PyModule_AddObject(module, "plShadowMaster", Init_pyShadowMaster_Type());
     PyModule_AddObject(module, "plPointShadowMaster", Init_pyPointShadowMaster_Type());
     PyModule_AddObject(module, "plDirectShadowMaster", Init_pyDirectShadowMaster_Type());
-    PyModule_AddObject(module, "pfGUICtrlProcObject", Init_pyGUICtrlProcObject_Type());
-    PyModule_AddObject(module, "pfGUIDialogProc", Init_pyGUIDialogProc_Type());
-    PyModule_AddObject(module, "pfGUICtrlProcWriteableObject", Init_pyGUICtrlProcWriteableObject_Type());
+    PyModule_AddObject(module, "pfGUIButtonMod", Init_pyGUIButtonMod_Type());
+    PyModule_AddObject(module, "pfGUICheckBoxCtrl", Init_pyGUICheckBoxCtrl_Type());
+    PyModule_AddObject(module, "pfGUIClickMapCtrl", Init_pyGUIClickMapCtrl_Type());
```

Comment:
It should be used on **all** PyObjects.  If you grep the sources for `PyType_Ready`, the only one should be in the definition of `PyType_CheckAndReady`.  I considered even poisoning the `PyType_Ready` name within PyPlasma.h, but decided against it at the last minute since 3rd party modules may #include PyPlasma.h, and I wouldn't want to break those.